### ISSUE TITLE
Touch pressing does not work when exporting to HTML5

### ISSUE THREAD

**Issue by NeDanels** [SpyrexDE/SmoothScroll#63]
When I export my game to HTML5, clicking on my TextureButtons stops working.

When running in Godot itself with mouse everything works fine, also when exporting to APK everything works fine.

When I took the button out of the SmoothScroolContainer it worked correctly and clicked even when exporting to HTML.

I attach the video, and the structure of my project.

This is the work on pc, here the button is pressed, and from hamster it says that there are not enough coins, everything works properly:

https://github.com/SpyrexDE/SmoothScroll/assets/76656925/b1415ba5-63e7-48a5-9aee-6197d932d544

This is a recording of the screen from the phone, the game works in the browser in HTML5, on TextureButton do not lead to anything

https://github.com/SpyrexDE/SmoothScroll/assets/76656925/0b86a10e-ae55-47b7-87c1-fe84731f5113

This is the structure of my TextureButton:
![image](https://github.com/SpyrexDE/SmoothScroll/assets/76656925/6112fc0c-88b7-4f1d-b7e8-957e4b9a62e7)

I tried to describe as much as possible, if there is any additional information I can provide, please write, I hope you can help me.

**Godot version**
Godot 3.5.3-stable

**Comment by SpyrexDE** [comment]
> When I took the button out of the SmoothScroolContainer it worked correctly and clicked even when exporting to HTML.

Does the issue persist when using a regular scroll container?

**Comment by NeDanels** [comment]
> > When I took the button out of the SmoothScroolContainer it worked correctly and clicked even when exporting to HTML.
> 
> Does the issue persist when using a regular scroll container?

No, used a regular scroll container, in the case of using it everything is fine.

**Comment by SpyrexDE** [comment]
Does it help when you remove [this line](https://github.com/SpyrexDE/SmoothScroll/blob/e5d17c8aaaa58ed5567dc61b5aafcba2a0ed8dee/addons/SmoothScroll/SmoothScrollContainer.gd#L291C2-L291C46)?
If not, could you please attach a minimal reproduction project for further investigation?

**Comment by NeDanels** [comment]
> Does it help when you remove [this line](https://github.com/SpyrexDE/SmoothScroll/blob/e5d17c8aaaa58ed5567dc61b5aafcba2a0ed8dee/addons/SmoothScroll/SmoothScrollContainer.gd#L291C2-L291C46)? If not, could you please attach a minimal reproduction project for further investigation?

Removed this line, the solution to the problem did not help, I will try in the next few days then throw the minimum working godot project.

**Comment by SpyrexDE** [comment]
If you still have issues with this, please upload a minimal reproduction project and I will reopen the issue.
